generators/ruby-v2/ast/src/ast/Class.ts

```diff
@@ -0,0 +1,86 @@
+import { Comment } from "./Comment";
+import { Module } from "./Module";
+import { AstNode } from "./core/AstNode";
+import { Writer } from "./core/Writer";
+
+export declare namespace Class_ {
+    export interface Args extends Module.Args {
+        /* The superclass of this class. */
+        superclass?: Class_;
+    }
+}
+
+export class Class_ extends Module {
+    public readonly superclass: Class_ | undefined;
+    public readonly statements: AstNode[];
```

Comment:
not blocking, but do you think it makes sense to have a separate argument for fields? Or is the consumer going to have to specify that as a statement some how? (something like `@@no_of_customers = 0`, stolen from [here](https://www.tutorialspoint.com/ruby/ruby_classes.htm))